clang/lib/3C/ArrayBoundsInferenceConsumer.cpp

```diff
@@ -106,7 +106,7 @@ static bool needNTArrayBounds(const ConstraintVariable *CV,
 
 static bool needArrayBounds(Expr *E, ProgramInfo &Info, ASTContext *C) {
   ConstraintResolver CR(Info, C);
-  CVarSet ConsVar = CR.getExprConstraintVars(E);
+  CVarSet ConsVar = CR.getExprConstraintVarsSet(E);
```

Comment:
Per my earlier comment: Hard to see why I should be calling `VarsSet` rather than `Vars` since both have the same signature. What distinguishes them?